java/bulk-export/bulk-export-core/src/main/java/sleeper/bulkexport/core/BulkExportQuerySplitter.java

```diff
@@ -149,6 +149,7 @@ protected List<String> getFiles(Partition partition) {
                 files.addAll(partitionToFiles.get(partitionId));
             }
         }
+        LOGGER.info("Found {} files for partition {}", files.size(), partition.getId());
```

Comment:
There are some logs similar to this in splitIntoLeafPartitionQueries. It might be worth putting the logs in the same method to make it more obvious how they'll work together?